langchain/llms/fake_llm.py

```diff
@@ -0,0 +1,31 @@
+"""Fake LLM wrapper for testing purposes."""
+from typing import Any, List, Mapping, Optional
+
+from pydantic import BaseModel
+
+from langchain.llms.base import LLM
+
+
+class FakeLLM(LLM, BaseModel):
+    """Fake LLM wrapper for testing purposes."""
+
+    queries: Optional[Mapping] = None
+    n: int = 1
```

Comment:
Added this because the FakeLLM in tests/unit_tests/chains/test_hyde.py had this.